make_matcher_backend/app/controllers/api/profiles_controller.rb

```
@@ -0,0 +1,31 @@
+class Api::ProfilesController < Api::Controller
+  before_action :authorized_user, :find_profile
+
+  def edit
+    render_profile
+  end
+
+  def update
+    @profile.update(profile_params)
```

`update!` will throw the exception, this way will currently swallow it right? perhaps we should throw it

---

Agreed